decred/decred/util/helpers.py

```diff
@@ -29,7 +29,7 @@ def formatTraceback(err):
         str: The __str__() of the error, followed by the standard formatting
             of the traceback on the following lines.
     """
-    return f"{err}\nTraceback:\n{traceback.print_tb(err.__traceback__)}"
+    return f"{err}\nTraceback:\n{traceback.extract_tb(err.__traceback__)}"
```

Comment:
`print_tb` prints to stderr autonomously and returns `None`. `extract_tb` generates the same output and actually returns it.

Reply:
It looks like, though, this has removed the formatting and the traceback isn't printed to stderr sometimes. Not sure exactly what's going on, but here's a screenshot when starting without internet:
![wierdlog](https://user-images.githubusercontent.com/34998433/79959760-77197b00-84bf-11ea-9aec-4e25791cdead.png)

Reply:
There is one instance of `app.appWindow.showError` in `AssetScreen` that sends the whole traceback to the window instead of just a short error message, fixing it.

Reply:
Addressing your other point about loss of traceback formatting.